samples/net/mdns_responder/src/service.c

```diff
@@ -66,7 +66,7 @@ void service(void)
 #else
```

Comment:
In commit `samples: net: mdns_responder: Fix coverity resource leak issue`
Same comment about commit message, please change to
```
Avoid resource leak by closing socket when there is an error.

Coverity-CID: 215381
Fixes #33094 

Signed-of-by: ...
```

Reply:
Fixed